scene/3d/path_3d.cpp

```diff
@@ -328,6 +344,7 @@ void PathFollow3D::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("set_tilt_enabled", "enabled"), &PathFollow3D::set_tilt_enabled);
 	ClassDB::bind_method(D_METHOD("is_tilt_enabled"), &PathFollow3D::is_tilt_enabled);
+	ClassDB::bind_method(D_METHOD("_update_transform"), &PathFollow3D::_update_transform);
```

Comment:
It is not necessary to bind to be able to call this deferred, you can use `callable_mp` AFAIK

Reply:
Right, it's also feels more straightforward. Thanks.